libraries/AP_RangeFinder/RangeFinder.cpp

```diff
@@ -164,11 +165,13 @@ const AP_Param::GroupInfo RangeFinder::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("_ORIENT", 53, RangeFinder, state[0].orientation, ROTATION_PITCH_270),
 
+    AP_SUBGROUPVARPTR(drivers[0], "_",  57, RangeFinder, backend_var_info[0]),
```

Comment:
I just remembered about an interesting problem, we won't have documentation for the parameters under this. I'm not sure how he can fix it.

Reply:
I've added a commit for the parameter docs